src/CheckIfDead.php

```diff
@@ -373,14 +469,14 @@ protected function getCurlOptions( $url, $full = false ) {
 			// Emulate a web browser request but make it accept more than a web browser
 			$header = [
 				// @codingStandardsIgnoreStart Line exceeds 100 characters
-				'Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5',
+				'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3',
```

Comment:
Although I'm not sure it matters, "image/apng" looks like a typo.

Reply:
This is what I copied out of Safari.  I honestly don't think it matters that much but we can change it.